hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -155,14 +156,16 @@ private void initNameNode(ExecutorDriver driver, String dnsName) throws Exceptio
         // is used as a mutex indicates whether or not any NameNode has ever been formatted.
         // The first NameNode to acquire the mutex and find that no NameNode has ever been
         // formatted, formats itself.  All others bootstrap.
```

Comment:
Ok it is clearer now. We should have updates to the comments. Also, please make use of constants, i.e. HDFSConstants.NN_STATUS_INIT_VAL.